dts/xtensa/espressif/esp32.dtsi

```diff
@@ -9,6 +9,7 @@
 #include <dt-bindings/i2c/i2c.h>
 #include <dt-bindings/clock/esp32_clock.h>
 #include <dt-bindings/interrupt-controller/esp-xtensa-intmux.h>
+#include <dt-bindings/pinctrl/esp32-pinctrl.h>
```

Comment:
I tried to build lvgl sample for odroid_go but this file is missing.

Reply:
@ydamigos I am facing no issues when building this sample for the odroid_go board. Did you try the following commands before?

```
west update
west espressif update
```

Reply:
False alarm. I always forget to run these commands first.

Reply:
Something similar was reported on Heltec's board a while ago. Could you try editing `boards/xtensa/odroid_go/board_init.c` and replace `SYS_INIT`'s second argument by `PRE_KERNEL_2` instead of `APPLICATION`?

Reply:
I did the change but it didn't work. I tried also `samples/drivers/display` with the same result.

Reply:
I agree., it's really strange. I checked odroid_go device tree files several times to find out what is wrong without success. I will try to create a Discord account the following days and ping you there.